client/src/components/screens/Patient/components/FilesTab/test/files.spec.tsx

```diff
@@ -0,0 +1,247 @@
+import React from 'react';
+import {
+    render, screen
+  } from '@testing-library/react';
+import AppTest from 'AppTest';
+
+import {getFilesData} from 'store/graphql/files/actions'
+import ApolloProvider from 'store/providers/apollo';
+
+import FilesTab from '..';
+
+jest.mock("store/graphql/files/actions");
+describe('PatientFiles', () => {
+
+  beforeEach(() => {
+    console.error = jest.fn();
+    (getFilesData as jest.Mock).mockReset();
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+    test('No files', async () => {
```

Comment:
try a better phrasing.

Reply:
les now des tests doivent expliquer le test. Generalement on commence : `test('shoud said there is no files when getFilesData in empty' ...)`